featurebyte/models/credential.py

```diff
@@ -226,8 +227,27 @@ class GCSStorageCredential(BaseStorageCredential):
     )
 
 
+class AzureBlobStorageCredential(BaseStorageCredential):
```

Comment:
do we plan on exposing this in the `__init__.py` too like the rest of the credentials?

https://github.com/featurebyte/featurebyte/blob/61fa75120943d94113730e8d40c0d9df18fc6204/featurebyte/__init__.py#L404-L408

if so, should also add to the `documentation_layout` under utility classes, and you'll probably also need to add it into this allowed classes list

https://github.com/featurebyte/featurebyte/blob/4cd3a28fad76f9472639f2d3451cd1786aaf31cc/featurebyte/common/documentation/allowed_classes.py#L8

Reply:
Yes. Thanks for pointing that out!